### PR TITLE
Adding code sniffer to the skeleton profile.

### PR DESCRIPTION
**Pull request by ordavidil**

### CODE REVIEW

File: app/templates/_travis.yml [hunk 1]
```diff
@@ -1,5 +1,15 @@
 language: php
 
+env:
+  - INSTALL_PROJECT=1
```
Comment (amitaibu) [inline app/templates/_travis.yml]: worth adding a comment here

**Comment by amitaibu** [comment]
Nice one!

**Comment by ordavidil** [comment]
@amitaibu 
ready for review

**Comment by amitaibu** [comment]
Published `v1.4.0`, thanks!

**Pull request merged**